src/integration/queries/JoinSessionQuery.graphql

```diff
@@ -39,6 +39,10 @@ query JoinSession($shortname: String!) {
     feedbacks {
       id
       content
+      tags {
```

Comment:
These tags should not be exposed to students (we would like to keep them internal).

As this file is used in the integration test, it might break the test if you change it in isolation.